community_supplied/Interface/snmp-interface-throughput.rule

```diff
@@ -1,11 +1,282 @@
+/*
+* Monitors interface throughput utilization and notifies when
+* anomalies are found.
+* Two input control detection
+*
+* 1) interface-name-variable, is a regular expression that matches
+* the interfaces that you would like to monitor. By default it '.*',
+* which matches all interfaces. Use something like 'ge.*' to
+* match only gigabit ethernet interfaces.
+*
+* 2) threshold, is the threshold that causes the rule to report
+* an anomaly. By default it is 75.00 %. This rule will set a
+* dashboard color to red when io throughput utilization percent
+* exceeds threshold value. Otherwise it is set to green.
+*/
 healthbot {
     topic interface.throughput {
         description "Rules that monitor interface throughput and react to changes";
         synopsis "Interface throughput analysis";
         rule check-interface-throughput-snmp {
             synopsis "Interface throughput threshold crossing";
             description "Monitors interface throughput and generates triggers on customizable threshold crossing";    
-	    	
+	    keys interface-name;
+            /*
+            * Monitors interface input and output throughput utilization. Notifies
+            * via the dashboard colors when interface io throughput utilization
+            * exceeds the threshold {{$threshold}} it sets the color to red.
+            * Otherwise the color is set to green.
+            *
+            * Use name as key for rule
+            */
+            function speed_conv {
+                path speed-conversion.py;
+                method speed_conv;
+                argument speed {
+                    mandatory;
+                }
+            }
+            /*
+            * Sensor configuration to get data from network devices.
+            */
+            sensor input-output-bytes {
+                synopsis "Interface SNMP sensor definition";
+                description "Interfaces SNMP sensor to data from network device";
+                snmp {
+                    table IF-MIB::ifXTable;
+                    frequency 60s;
```

Comment:
Increase frequency from 60s to 180s

Reply:
Addressed and Increased frequency from 60s to 180s